### PR TITLE
Update logging from auditd module

### PR DESCRIPTION
**Pull request by andrewkroh**
Use `logp.Logger` for all logging from the auditd module. I also increased the logging level for two statements because they will be useful for troubleshooting (without having to ask users to re-run with debug enabled).

### CODE REVIEW

File: auditbeat/module/auditd/audit_linux.go [hunk 1]
```diff
@@ -116,14 +115,14 @@ func (ms *MetricSet) Run(reporter mb.PushReporterV2) {
...
-		logp.Err("%v %v", logPrefix, err)
+		ms.log.Errorw("Failure adding audit rules", "error", err)
 		return
 	}
 
 	out, err := ms.receiveEvents(reporter.Done())
 	if err != nil {
 		reporter.Error(err)
-		logp.Err("%v %v", logPrefix, err)
+		ms.log.Errorw("Failure receiving audit events", "error", err)
```
Comment (ruflin) [inline auditbeat/module/auditd/audit_linux.go]: Would be nice if this would be `error.message` the same as we have in our `fields.yml`.

File: metricbeat/mb/testing/modules.go [hunk 2]
```diff
@@ -269,7 +269,10 @@ func RunPushMetricSetV2(timeout time.Duration, waitEvents int, metricSet mb.Push
 			select {
 			case <-timer.C:
 				return
-			case e := <-r.eventsC:
+			case e, ok := <-r.eventsC:
```
Comment (ruflin) [inline metricbeat/mb/testing/modules.go]: @andrewkroh Is this change related? LGTM but curious if that is related to an other issue.

Comment (andrewkroh) [reply metricbeat/mb/testing/modules.go]: While testing some failure conditions I noticed the test case didn’t stop immediately and this was the fix.

File: auditbeat/module/auditd/audit_linux.go (hunk 1 above)
Comment (andrewkroh) [reply auditbeat/module/auditd/audit_linux.go]: It would. The error is written as an object, but it doesn’t match our normal structure. See https://github.com/uber-go/zap/blob/master/zapcore/error.go#L28

Comment (andrewkroh) [reply auditbeat/module/auditd/audit_linux.go]: It should be possible to intercept (but with some expense) the Error types and wrap them in another type that implements https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler to customize the marshaling.

Comment (ruflin) [reply auditbeat/module/auditd/audit_linux.go]: The part I'm worried if we don't do it is that someone is going to read filebeat json logs which will conflict with our own filebeat template.
